presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -461,6 +465,21 @@ class SystemConfig : public ConfigBase {
   int32_t oldTaskCleanUpMs() const;
 };
 
+/// Provides access to connector properties defined in
+/// catalog/$connector_name.properties file.
+class ConnectorConfig : public ConfigBase {
+ public:
+  static constexpr std::string_view kConnectorName{"connector.name"};
+  static constexpr std::string_view kHiveImmutablePartitions{
```

Comment:
We today have the TPC-H connector and potentially other connectors in the future. Why are we adding Hive Connector specific fields here?

Reply:
That's a good point, then I think the default value can be set in velox HiveConfig instead.

Will have this PR to solve the issue: https://github.com/facebookincubator/velox/pull/6658